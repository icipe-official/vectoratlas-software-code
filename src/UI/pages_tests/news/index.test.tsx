import React from 'react';
import NewsPage from '../../pages/news/index';
import { render } from '../../test_config/render';

jest.mock(
  '../../components/news/newsList',
  () =>
    function NewsListMock() {
      return <div>News list mock</div>;
    }
);

describe('news page', () => {
  it('renders correctly', () => {
    const { wrapper } = render(<NewsPage />);

    expect(wrapper.container).toMatchSnapshot();
  });
});
