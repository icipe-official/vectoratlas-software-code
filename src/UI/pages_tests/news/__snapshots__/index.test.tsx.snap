// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`news page renders correctly 1`] = `
<div>
  <div>
    <main>
      <div
        class="MuiContainer-root css-j5ise1-MuiContainer-root"
      >
        <div>
          News list mock
        </div>
      </div>
    </main>
  </div>
</div>
`;
